apps/storybook/src/TeamGroupCard.stories.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import { number, text } from '@storybook/addon-knobs';
+
+import { TeamGroupCard } from '@asap-hub/react-components';
+import {
+  createListGroupResponse,
+  createTeamResponse,
+} from '@asap-hub/fixtures';
+import { ListGroupResponse } from '@asap-hub/model';
+
+export default {
+  title: 'Organisms / Team Profile/ Group Card',
+  component: TeamGroupCard,
+};
+
+const props = (): ListGroupResponse => {
+  const numberOfGroups = number('Number of groups', 5);
+  let groups = createListGroupResponse(numberOfGroups);
```

Comment:
I think looping and using `createGroupResponse` like you already do down there with `createTeamResponse` could save some boilerplate?